clients/discord-bot/src/commands/administrator-commands/debug-commands.marshal.ts

```diff
@@ -80,9 +87,13 @@ export class DebugCommandsMarshal extends Marshal {
                     collectorType: WizardType.REACTION,
                     collectorTarget: response,
                 },
-                instructions: "React to the Wizard Debugger to throw an error within the Wizard Step.",
+                instructions:
+                    "React to the Wizard Debugger to throw an error within the Wizard Step.",
```

Comment:
Is there a max line length or something at play here? Seeing a lot of changes like this that don't seem like a great threshold

Reply:
Fixed
https://github.com/SprocketBot/sprocket/pull/327/commits/973a46d75de2031f225fff5fab8f1953d63f4976